### PR TITLE
274 add support for solidly uni v2 style pools

### PR DESCRIPTION
**Pull request by Lesigh-3100**

### CODE REVIEW

File: fastlane_bot/data/blockchain_data/ethereum/tokens.csv
Comment (mikewcasale) [inline fastlane_bot/data/blockchain_data/ethereum/tokens.csv:1]: is there a reason we are adding more tokens with the code changes?

File: fastlane_bot/data/blockchain_data/coinbase_base/uniswap_v3_event_mappings.csv
Comment (mikewcasale) [inline fastlane_bot/data/blockchain_data/coinbase_base/uniswap_v3_event_mappings.csv:1]: similarly with tokens.csv, are these additions needed to support the code changes?

File: fastlane_bot/data/blockchain_data/coinbase_base/uniswap_v2_event_mappings.csv
Comment (mikewcasale) [inline fastlane_bot/data/blockchain_data/coinbase_base/uniswap_v2_event_mappings.csv:1]: similarly with tokens.csv, are these additions needed to support the code changes?

File: fastlane_bot/data/blockchain_data/coinbase_base/tokens.csv
Comment (mikewcasale) [inline fastlane_bot/data/blockchain_data/coinbase_base/tokens.csv:1]: similarly with tokens.csv, are these additions needed to support the code changes?

File: fastlane_bot/data/blockchain_data/coinbase_base/static_pool_data.csv
Comment (mikewcasale) [inline fastlane_bot/data/blockchain_data/coinbase_base/static_pool_data.csv:1]: similarly with tokens.csv, are these additions needed to support the code changes?

**Review by mikewcasale — CHANGES_REQUESTED** [review]
Please only include necessary changes to the csv files so that we don't inadvertently introduce errors there.

File: fastlane_bot/data/blockchain_data/ethereum/tokens.csv
Comment (Lesigh-3100) [reply fastlane_bot/data/blockchain_data/ethereum/tokens.csv:1]: No - this is not necessary

File: fastlane_bot/data/blockchain_data/coinbase_base/static_pool_data.csv
Comment (Lesigh-3100) [reply fastlane_bot/data/blockchain_data/coinbase_base/static_pool_data.csv:1]: This specifically does need to be added - yes. There were significant important changes to the static data on Base.

File: fastlane_bot/data/blockchain_data/coinbase_base/tokens.csv
Comment (Lesigh-3100) [reply fastlane_bot/data/blockchain_data/coinbase_base/tokens.csv:1]: Not entirely necessary - could be removed

File: fastlane_bot/data/blockchain_data/coinbase_base/uniswap_v2_event_mappings.csv
Comment (Lesigh-3100) [reply fastlane_bot/data/blockchain_data/coinbase_base/uniswap_v2_event_mappings.csv:1]: This should be included as it adds more pools available to trade on. Not a -requirement- of the code to operate, but important.

File: fastlane_bot/data/blockchain_data/coinbase_base/uniswap_v3_event_mappings.csv
Comment (Lesigh-3100) [reply fastlane_bot/data/blockchain_data/coinbase_base/uniswap_v3_event_mappings.csv:1]: This should be included as it adds more pools available to trade on. Not a -requirement- of the code to operate, but important.

**Review by mikewcasale — APPROVED** [review]
Looks good

**Pull request merged**